paths/electron/packaging-apps/02-using-electron-packager.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Use electron-packager
+permalink: /electron/package-your-app/use-electron-packager/
+next-page: /electron/package-your-app/add-an-icon/
+facilitator: false
+sidebar:
+  nav: "electron-package-your-app"
+main-content: |
+
+  Now that we have installed `electron-packager`, we're ready to use it. We could type out the entire command every time, but it's best practice to add a build script to the `package.json` file. This means that every time we run the default build script, our specific list of commands will be run.
+
+  ![gif of following the directions below](../../../images/gifs/electron/electron2-packaging.gif)
+
+
+  1. In the `package.json` file, add the following script in "scripts" of package.json.
+
+          "build": "electron-packager . electron-app --all --overwrite --ignore=node_modules/electron-*"
```

Comment:
@malept Ah, good to know. I included those based on the tutorials I followed when I was first using electron. Thank you for the feedback, I will remove the ignore parameter. 👍  

For the name in the command line, I think it will be easier for this tutorial since we don't go into too much detail about the package.json file. Do you think that's OK?